chatrooms.lua

```diff
@@ -9,10 +9,14 @@
 
 function OnStartup()
 	tConfig = {
+		iMaxHistory = 35,
 		sGlobalPath = "/www/ChatLogs/",
+		sPath = Core.GetPtokaXPath().."scripts/",
+		sExtPath = "external/",
 		iTotalProfiles = table.getn( ProfMan.GetProfiles() ),
```

Comment:
`table.getn` etc. were removed in [Lua 5.2](http://www.lua.org/manual/5.2/manual.html#6.5). Use the __len metamethod (called by using `#` or `select`)